homeassistant/components/ps4/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Support for PlayStation 4 consoles.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/ps4/
+"""
+import logging
+
+from homeassistant.components.ps4.config_flow import PlayStation4FlowHandler
+from homeassistant.components.ps4.const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['pyps4-homeassistant==0.2.6']
+
+
+async def async_setup(hass, config):
+    """Set up the PS4 Component."""
+    _LOGGER.debug("Registering PlayStation 4 Config Flow Version %s",
+                  PlayStation4FlowHandler.VERSION)
+    hass.data[DOMAIN] = {}
```

Comment:
No reason of doing this here, weird dependency since you don't really need this function at all.

Reply:
I know. When testing after I separated the config_flow, I get an exception for flow not found unless I imported the flow explicitly. Perhaps Ill retry after rebase.

Reply:
We have to import the flow into the component to make sure the flow module is read and the flow registered. We can disable the pylint/flake8 warning about not used import on that line.

Reply:
Oh. Ok. Removed that line.

Reply:
Everything should be ready now.

Reply:
This isn't used, so we can remove this line.